lib/demo-data.js

```
@@ -0,0 +1,120 @@
+// Demo data for pr-vibe showcase - no authentication required
+
+export const demoPRData = {
+  number: 42,
+  title: "Add user authentication system",
```

ESLint

[error] Strings must use singlequote.

(quotes)